Integrations/Integration-Wildfire.yml

```diff
@@ -148,19 +131,36 @@ script:
               return 'Something went wrong.\n Status code: ' + res.StatusCode + '\nBody: '+ JSON.stringify(res) + '.';
           break;
 
-      case 'WildFireReport':
+      case 'wildfire-report':
         url += urlDict.report;
-        var hash = args.md5 || args.sha256;
+        var hash = args.md5 || args.hash;
         if(!hash)
-            throw 'Missing md5/sha256';
+            throw 'Missing md5/hash argument.';
 
         var bodyXML = 'apikey='+params.token+'&format=xml&hash='+hash;
         var resXML = sendRequest(url, bodyXML, headers);
         var result = JSON.parse(x2j(resXML));
         var report = result.wildfire.task_info.report;
```

Comment:
Please use `dq` to access nested elements - otherwise it will throw an error if the element does not exist